packages/typespec-test/test/loadtesting_modular/generated/typespec-ts/package.json

```diff
@@ -51,8 +57,6 @@
     "@azure/core-rest-pipeline": "^1.5.0",
     "@azure/logger": "^1.0.0",
     "tslib": "^2.6.2",
-    "@azure/core-lro": "3.0.0",
```

Comment:
why is this being removed? Have they removed lro operation ?

Reply:
I believe so, in old spec they defined as customized lro but finally removed out. I checked with service team with below message.

![image](https://github.com/Azure/autorest.typescript/assets/9943211/7135ca6b-28d2-4ee8-8c19-92e0a136f3d2)